app/components/articleSearch/components/searchItem/authors.tsx

```diff
@@ -39,56 +40,65 @@ function getAuthorOrganization(organization: string) {
   }
 }
 
-const Authors = (props: IAuthorsProps) => {
-  if (props.authors.size <= 3) {
-    const authorItems = props.authors.map((author, index) => {
+const Authors = ({ authors, isAuthorsOpen, toggleAuthors }: IAuthorsProps) => {
+  const MINIMUM_SHOWING_AUTHOR_NUMBER = 3;
```

Comment:
a constant statement should be positioned at top of the file.(should live in global scope) just below of the import statements.